content.md

```diff
@@ -40,10 +49,16 @@ This list is also available online at: https://resources.osucord.moe/
 ### Training Tools
 - [osu!Trainer](https://github.com/FunOrange/osu-trainer) - A program that allows you to modify the difficulty of a beatmap very quickly and easily
 - [cosu!Trainer](https://github.com/hwsmm/cosutrainer) - osu!Trainer alternative for Linux, written in C
+- [Shikke's Anti-Mindblock](https://shikkesora.com/downloads) - A tool to quickly flip your monitor upside-down to help with mind-block
 ---
-### Self-improvement
+### Guides
 - [khz's streaming guide](https://docs.google.com/document/d/1xbNwH_vN4O3azBYwua-Z_3GOAI2d-kV_EeE96aiRLBU/edit) - A guide on how to get better at streaming
-- [Shikke's Anti-Mindblock](https://shikkesora.com/downloads) - A tool to quickly flip your monitor upside-down to help with mind-block
+- [osu! phd](https://www.youtube.com/watch?v=uc99yWeP1h4) - A comprehensive introduction to every skill in the game and how they relate to each other
+- [verto's osu!guides](https://www.youtube.com/playlist?list=PLGeq_WL32alSbhg0GgL_a_Dj6t74Y9M33) - Explanations and discussions of osu! terms in video form
+- [Spare's guide to improve tapping](https://www.youtube.com/watch?v=yXdtbFNgCfY) - Tapping tips directly from a top 100 player
+- Audio latency improvement guides:
+  - [For Windows, by g00burr](https://www.reddit.com/r/osugame/comments/hqoakj/tutorial_removing_perceivable_latency_for_good/)
+  - [For Linux, by ThePoon](https://blog.thepoon.fr/osuLinuxAudioLatency/)
```

Comment:
The windows guide is probably fine (except for my personal hate for REAL because it can lead to random sized audio buffers between restarts of the application)

The linux resource however is just straight 6 years out of date and talks about patches that don't even apply to wine anymore. The "current" patch is much different to this and is less influential now (though still does actually drop latency)

Just about the only up to date thing there is the osu-wine AUR package and that is maintained separately to this blog post. (wine 3.10 is ancient)

The goto solution for installing osu on Linux already includes a patched wine, and gets updates (https://github.com/NelloKudo/osu-winello) but it doesn't handle the system side of audio configurations.

Reply:
Is there a better resource we could link to, or should it just be removed?

Reply:
Windows resource can stay, Linux one should be removed imo, Pipewire already has pretty good out of the box latency and depending on your soundcard messing with it can lead to some unstable results.

There aren't that many good Linux audio latency based resources that are easily digestable. I wouldn't expect the average user to want to mess with config files and testing to make sure their changes are actually stable. (additionally there is just too many factors and soundservers to consider).